src/main/java/at/hannibal2/skyhanni/data/PetAPI.kt

```diff
@@ -35,31 +194,412 @@ object PetAPI {
         "➡",
     )
 
+    @Deprecated(message = "use PetAPI.inPetMenu")
```

Comment:
inPetMenu might be invalid when using isPetMenu inside a feature. so this is currently no valid alterantive.